data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1701 @@
+		not "Heliarch License: done"
+	to fail
+		or
+			has "Heliarch Investigation 2: active"
+			has "Heliarch License: done"
+			"reputation: Quarg" < 0
+	on offer
+		require "Jump Drive"
+		conversation
+			`As you prepare to land on <origin>, your monitor starts producing a low hum, stops, then starts again. This keeps up for nearly a minute until a message pops up. "Captain <last>, come to trust you our comrades have, thanks to your continuous help to our cause. Interested we are in your continuous support, so that free the Coalition, together we can. To <destination> you must come, meet with you, our leaders will, so that discuss an important task with them, you may. An available bunk in your ship, you should have."`
+			`	The message cuts off shortly after you finish reading it. Your monitor is back to normal, and no humming sound can be heard. You mark <planet> on your map.`
+				accept
+
+mission "Resistance: Quarg Interview"
+	name "The Interview"
+	description "Travel to <destination> with Oobat, an Arach diplomat from the Lunarium, to try and interview the Quarg about their thoughts on the Coalition."
+	passengers 1
+	landing
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	source "Remote Blue"
+	destination "Lagrange"
+	to offer
+		has "Resistance: Questions: done"
+	on offer
+		conversation
+			`You leave your ship to find a pair of Kimek waiting for you. "Follow us to the meeting point, you will," one of them says, and you oblige, trying to keep a low profile as you go with them across the city. You reach a district near the center of town, with modest blocks and few skyscrapers, and head into what seems to be a Kimek restaurant. You continue following the two Kimek past busy tables and Kimek patrons who all have eyes on you as you head to the back.`
+			`	Taking an elevator, you move down a few levels. You arrive at some doors that open to an Arach delegation, with half a dozen bulky Arachi surrounding a young, fine-dressed lady in a blue "dress" with white lines wrapping around it. "Oh, finally arrived, have they?" she says as you step off the elevator, coming to meet you. "<first> <last>, is it? So intrigued by our most recent recruitee, I was, that decided I did to come to this meeting for once. Lady Lobmab Bebliss, I am, head of our great house, and benefactor to the Lunarium." She steps one leg forward, to you, as if in some archaic Arach gesture. Unsure on how to respond, you do your best pompous bow. She seems satisfied, bringing her leg back. "Confess I must, however, that most incovenient it is, to come so far away while maintaining a low profile. Difficult to just disappear, it is, when the head of the greatest Arach House, you are. Hope I do that worth it, your contributions are."`
+			choice
+				`	"I'll do what I can to help."`
+					goto leaders
+				`	"You speak as if you've done a whole lot than me to help the Lunarium."`
```

Comment:
I tried to have House Bebliss' head come out as somewhat obnoxious, with acting high and mighty. This choice was to let the player say something like "we've got a badass over here," only to have Lobmab just not care and confirm with "well observed, I'm a badass."
Only, without recurring to the meme wording itself.

I see what you mean about the wording being too hostile though, yeah.

Reply:
Ohhhh that makes more sense. I definitely got the high and mighty feel from her as I was reading her lines, so spot on!

I think her "yes, well observed" comment was in reference to First Last saying that they observed that she's contributed a lot to the Lunarium cause, which she has, as she elaborates on in the next part about her family helping. Like a "yes, well-observed that you noticed that I have done so much to help the cause." I feel like it keeps her high and mighty attitude since she isn't being humble whatsoever. Thoughts?